Source/JavaScriptCore/bytecode/ObjectPropertyCondition.h

```diff
@@ -120,7 +120,22 @@ class ObjectPropertyCondition {
             vm.writeBarrier(owner);
         return absenceOfSetEffectWithoutBarrier(object, uid, prototype);
     }
-    
+
+    static ObjectPropertyCondition absenceOfIndexedWithoutBarrier(JSObject* object, JSObject* prototype)
```

Comment:
name nit: Can we call this absenceOfIndexedProperties in all the various function/enum names?

Reply:
Changed.